rdopkg_tar/tar_changes.py

```diff
@@ -123,6 +123,14 @@ def commit_distgit_amend(suffix):
     cmd = ['commit', '-a', '-F', '-', '--amend']
     git(*cmd, input=message, print_output=True)
 
+def check_gitlabuser():
+    # Check for the original commiter username in jenkins env variables.
+    # If it exists, then amend a suffix to the commit.
+    userName = os.environ.get('gitlabUserName')
+    gitlabuserName = os.environ.get('gitlabUserUsername')
+
+    if userName or gitlabuserName:
+        commit_distgit_amend(suffix='GitLab-User: ' + gitlabuserName + ' (' + userName + ')')
```

Comment:
One of these two commits changes this line from double-quotes `"` to `'` single-quotes. Please amend that commit to avoid this because it does not relate to the commit. "One (logical) change per commit, one commit per change." is a lot simpler and easier to review.